e2e/tests/gke-apps/cromwell.spec.ts

```diff
@@ -0,0 +1,93 @@
+import { findOrCreateWorkspace, signInWithAccessToken } from 'utils/test-utils';
+import AppsPanel from 'app/sidebar/apps-panel';
+import Button from 'app/element/button';
+import CromwellConfigurationPanel from 'app/sidebar/cromwell-configuration-panel';
+import BaseElement from 'app/element/base-element';
+import { waitForFn } from 'utils/waits-utils';
+
+// 10 minutes
+jest.setTimeout(10 * 60 * 1000);
+
+describe('Cromwell GKE App', () => {
+  beforeEach(async () => {
+    await signInWithAccessToken(page);
+  });
+
+  const workspaceName = 'e2eCreateCromwellGkeAppTest';
+
+  test('Create and delete a Cromwell GKE app', async () => {
+    await findOrCreateWorkspace(page, { workspaceName });
+
+    const appsPanel = new AppsPanel(page);
+    await appsPanel.open();
+
+    // Cromwell is not running, so it appears in unexpanded mode
+
+    const unexpandedCromwellXPath = `${appsPanel.getXpath()}//*[@data-test-id="Cromwell-unexpanded"]`;
+    const unexpandedCromwell = new Button(page, unexpandedCromwellXPath);
+
+    expect(await unexpandedCromwell.exists()).toBeTruthy();
+    await unexpandedCromwell.click();
+
+    // clicking an unexpanded GKE App:
```

Comment:
these tests are slightly different, due to current behavior differences between the 2 apps.